src/api/FileActivities.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * @flow
+ * @file Helper for the box File Activity API
+ * @author Box
+ */
+
+import Base from './Base';
+import { PERMISSION_CAN_COMMENT, PERMISSION_CAN_VIEW_ANNOTATIONS, ERROR_CODE_FETCH_ACTIVITY } from '../constants';
+import type { BoxItemPermission } from '../common/types/core';
+import type { ElementsXhrError } from '../common/types/api';
+import type { FileActivity, FileActivityTypes } from '../common/types/feed';
+
+// We only show the latest reply in the UI
+const REPLY_LIMIT = 1;
+
+const getFileActivityQueryParams = (
+    fileID: string,
+    activityTypes?: FileActivityTypes[] = [],
+    enableReplies?: boolean = true,
```

Comment:
is the `enableReplies` parameter needed? all usages in this file are `true` and the function is not exported